docs/data.json

```diff
@@ -239,6 +239,46 @@
+      "nvidia_pytorch": "Not applicable",
+      "nvidia_tensorflow": "Not applicable",
+      "nvtabular": "23.2.0",
+      "openmpi": "4.1.4",
+      "os": "Ubuntu 20.04.5 LTS",
+      "python_major": "3",
+      "pytorch": "Not applicable",
+      "release": "23.02",
+      "rmm": "22.08.00a+62.gf6bf047.dirty",
+      "size": "780.56 GB",
+      "sm": "60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80, 60, 61, 70, 75, 80",
```

Comment:
There's a bug here, but the data isn't presented to customers.  My $0.02 is to merge as is and update the extractor when more convenient.